DAS/README.md

```diff
@@ -3,17 +3,35 @@
 *This directory contains scripts to log and test various components of the data-acquisition-system*
 
 
+- [DAS](#das)
+  - [Basic Setup and Usage](#basic-setup-and-usage)
+  - [Unit Tests](#unit-tests)
+  - [V3 MQTT Recorder](#v3-mqtt-recorder)
+    - [Usage](#usage)
+  - [V3 MQTT Playback](#v3-mqtt-playback)
+    - [Usage](#usage-1)
+  - [V3 Fake Module](#v3-fake-module)
+    - [Usage](#usage-2)
+  - [V2 MQTT Playback](#v2-mqtt-playback)
+    - [Usage](#usage-3)
+
 
 ## Basic Setup and Usage
-These steps assume that you are in the data-acquisition-system root directory and poetry installed.
-1. `cd DAS`
-2. `poetry install`
-3. `poetry shell`
```

Comment:
I feel it may be helpful to still mention needing to run `poetry shell` to enter the virtual environment. What do u think?

Reply:
You don't need to enter the virtual environment if you run `poetry run`. I think it is much simpler for newer users of das to run a single `poetry install` than just `poetry run` every time.

I will add an optional `poetry shell` to `basic setup and usage`.